.github/actions/percy-snapshot/action.yml

```diff
@@ -27,12 +27,12 @@ runs:
   using: composite
   steps:
     - name: Install Python
-      uses: actions/setup-python@v5.1.0
+      uses: actions/setup-python@v5.1.1
       with:
         python-version: 3.10.14
 
     - name: Install NodeJS
-      uses: actions/setup-node@v4.0.2
+      uses: actions/setup-node@v4.0.3
```

Comment:
These actions dependencies are also not tested by our CI, but they shouldn't be a problem.